docs/structure-of-a-contract.rst

```diff
@@ -54,6 +54,66 @@ EVM Version
 
 The EVM version can be set with the ``evm-version`` pragma, which is documented in :ref:`evm-version`.
 
+Imports
+=======
+
+Import statements allow you to import :ref:`modules` or :ref:`interfaces` with the ``import`` or ``from ... import`` syntax.
+
+Imports via ``import``
+----------------------
+
+You may import modules (defined in ``.vy`` files) and interfaces (defined in ``.vyi`` or ``.json`` files) via ``import`` statements. You may use plain or ``as`` variants.
+
+.. code-block:: vyper
+
+    # without an alias
+    import my_package.foo
+
+    # with an alias
+    import my_package.foo as bar
+
+Imports via ``from ... import``
+-------------------------------
+
+Using ``from`` you can perform both absolute and relative imports. You may optionally include an alias - if you do not, the name of the interface will be the same as the file.
+
+.. code-block:: vyper
+
+    # without an alias
+    from my_package import foo
+
+    # with an alias
+    from my_package import foo as bar
+
+Relative imports are possible by prepending dots to the contract name. A single leading dot indicates a relative import starting with the current package. Two leading dots indicate a relative import from the parent of the current package:
```

Comment:
"Two leading dots indicate a relative import from the parent of the current package." Also say that the logic carries over to further higher directories so people know they can use `...` etc as well.